Dashboard/Views/StatusView.axaml

```diff
@@ -45,211 +45,399 @@
+                                            <TextBlock Text="{Binding K3State}" FontSize="22"
+                                                       Grid.Column="1" VerticalAlignment="Center" Background="White"
+                                                       Name="K3State" Margin="10, 0, 10, 0" Padding="10,0" TextAlignment="Center" />
+                                        </Grid>
+                                    </StackPanel>
+                                    <StackPanel Grid.Row="2" Grid.Column="0" Margin="0,10,0,0">
+                                        <Grid ColumnDefinitions="Auto, *">
+                                            <TextBlock FontSize="24" Text="RRQ:" />
+                                            <TextBlock Text="{Binding ResRadioQuality}" FontSize="22"
+                                                       Grid.Column="1" VerticalAlignment="Center" Background="White"
+                                                       Name="ResRadioQuality" Margin="10, 0, 10, 0" Padding="10,0" TextAlignment="Center" />
+                                        </Grid>
+                                    </StackPanel>
+                                    <StackPanel Grid.Row="3" Grid.Column="0" Margin="0,10,0,0">
+                                        <Grid ColumnDefinitions="Auto, *">
+                                            <TextBlock FontSize="24" Text="Node ID:" />
+                                            <TextBlock Text="{Binding ResNodeId}" FontSize="22"
+                                                       Grid.Column="1" VerticalAlignment="Center" Background="White"
+                                                       Name="ResNodeId" Margin="10, 0, 10, 0" Padding="10,0" TextAlignment="Center" />
+                                        </Grid>
+                                    </StackPanel>
+                                </Grid>
+                                <!-- Remote Emergency Stop Status -->
+                                <StackPanel HorizontalAlignment="Right" VerticalAlignment="Bottom" Margin="0,80,70,10">
+                                    <TextBlock Text="RES Status" FontSize="24" VerticalAlignment="Bottom"
+                                               HorizontalAlignment="Center" />
+                                    <Image Name="ResTabIcon" Source="../Assets/emergency-stop.png" Width="200"
+                                           Height="200" Opacity="1" />
+                                    <TextBlock Name="ResTab" FontWeight="Bold" Text="{Binding RemoteEmergency}"
+                                               FontSize="22" Background="Green" Foreground="White" Padding="10,0"
+                                               Margin="0,0,0,10" TextAlignment="Center" HorizontalAlignment="Center"
+                                               Width="200" />
+                                </StackPanel>
+                            </StackPanel>
```

Comment:
may be able to add another grid on the page for this so that the icon is always in the bottom right corner